docker-envs/base-ror-docker/es/install-ror-es-using-file.sh

```diff
@@ -13,6 +13,6 @@ echo "Installing ES ROR from file..."
 /usr/share/elasticsearch/bin/elasticsearch-plugin install --batch file:///tmp/ror.zip
 if verlte "6.5.0" "$ES_VERSION"; then
   echo "Patching ES ROR..."
-  /usr/share/elasticsearch/jdk/bin/java -jar /usr/share/elasticsearch/plugins/readonlyrest/ror-tools.jar patch --I_UNDERSTAND_AND_ACCEPT_ES_PATCHING yes
+  /usr/share/elasticsearch/jdk/bin/java -jar /usr/share/elasticsearch/plugins/readonlyrest/ror-tools.jar patch --I_UNDERSTAND_AND_ACCEPT_ES_PATCHING=yes
```

Comment:
I see the scripts can use old versions of ROR (like it's done in the ROR Sandbox)
If so, we have to be backward compatible here. Please try to run the stack for ROR 1.62.0 ;)

Reply:
I added an additional check (if) for ROR version. The new flag is added for versions >=1.64 (taking into account -pre versions). Tested with 1.62 and 1.64 and with ES 8.13. Was indeed failing before this change, now the container is starting.

Reply:
And today added additional adjustments for testing with ROR zip file or from sources - it required passing the version env too, same change as in ror-sandbox.

Reply:
I applied the changes from https://github.com/beshu-tech/ror-sandbox/pull/67#discussion_r2030191735 (the docker example in this repo is similar to ror-sandbox, so the comment from there apllied also here)